### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by alisa-bielenko**
- [DEMO LINK](https://alisa-bielenko.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://alisa-bielenko.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://alisa-bielenko.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://alisa-bielenko.github.io/layout_product-cards/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:4]: Add these links to PR description

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,52 @@
...
+        <p class="card__price-value">$2,199</p>
+      </div>
+
+      <a
+        href="#buy"
+        class="card__buy"
+        data-qa="hover"
+      >
+      Buy
+      </a>
```
Comment (volodymyr-soltys97) [inline src/index.html:53]:
```suggestion
      >
        Buy
      </a>
```

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    font-size: 12px;
+    line-height: 18px;
+    color: $second-text-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/card.scss:53]: You can write these styles in the selector above and nest them inside

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's make your code more perfect

File: readme.md (hunk 1 above)
Comment (alisa-bielenko) [reply readme.md:4]: ok

**Review by mykhalenych — CHANGES_REQUESTED** [review]
check your test link it isn't working

**Pull request closed without merging**